runtime/src/bank.rs

```diff
@@ -3443,6 +3444,8 @@ impl Bank {
             !self
                 .feature_set
                 .is_active(&remove_deprecated_request_unit_ix::id()),
+            self.feature_set
```

Comment:
Outside the scope of this review/PR: I'm slightly concerned with the fees being calculated in different places w/ (potentially) different feature bools passed in, it seems like we should have a single call point that handles all the feature activiation stuff wrapped for us. i.e. `Bank::calculate_fee` just takes a `feature_set` instead of these bools.

Reply:
Concern isn't with this PR, but moreso with us restructuring fees, adding fee bins, etc.

Reply:
Thanks for raising the concern. A possible counter argument is `calculate_fee` doesn't need to depend on `feature_set`, in some cases, `feature_set` doesn't even exist when calculating fee.